src/shared/services/form/form-value.service.ts

```diff
@@ -6,14 +6,12 @@ import { FieldTypeSanitiser } from './field-type-sanitiser';
 
 @Injectable()
 export class FormValueService {
-  public static readonly LABEL_SUFFIX = '-LABEL';
-
   /**
    * Gets value of a field based on fieldKey which is a dot separated reference to value and collection index.
    * There are two exeptions:
-   * 1) In case of a multiselect being identified as a leaf a '-LABEL' suffix is appended to the key and values og that key are returned
+   * 1) In case of a multiselect being identified as a leaf a '---LABEL' suffix is appended to the key and values og that key are returned
```

Comment:
@paul-graham - a nit-pick, I `og`led the comment and suspect it's meant to be `of`, instead 😄

Reply:
Changed. 🙂